opcua/common/uaerrors/_auto.py

```diff
@@ -0,0 +1,596 @@
+#AUTOGENERATED!!!
```

Comment:
I do not feel strongly one way or another. I think we could argue that in the other cases we have a different module layout:

`ua` imports `ua.uaprotocol_auto` and `ua.uaprotocol_hand`, in this case there is no parent module that imports both, so we'd end up with 3 siblings:

   ua.common.uaerrors
   ua.common._uaerrors_hand
   ua.common._uaerrors_auto